interfaces/cython/cantera/units/solution.py.in

```diff
@@ -0,0 +1,70 @@
+from .. import Solution as _Solution, PureFluid as _PureFluid, Heptane as _Heptane, CarbonDioxide as _CarbonDioxide, Hfc134a as _Hfc134a, Hydrogen as _Hydrogen, Methane as _Methane, Nitrogen as _Nitrogen, Oxygen as _Oxygen, Water as _Water
+import numpy as np 
+from pint import UnitRegistry
+units = UnitRegistry()
+Q_ = units.Quantity
+
+__all__ = ("units", "Q_", "Solution", "PureFluid", "Heptane", "CarbonDioxide","Hfc134a", "Hydrogen", "Methane", "Nitrogen", "Oxygen", "Water")
```

Comment:
As with the code in SConscript, make sure that this is limited to 80-90 characters.